modular_skyrat/modules/goofsec/code/department_guards.dm

```diff
@@ -296,13 +297,15 @@
 	ears = /obj/item/radio/headset/headset_med
 	uniform = /obj/item/clothing/under/rank/security/officer/blueshirt/skyrat/orderly
 	shoes = /obj/item/clothing/shoes/sneakers/white
-	head =  /obj/item/clothing/head/helmet/blueshirt/skyrat/guard
+	head =  /obj/item/clothing/head/beret/sec/medical
 	suit = /obj/item/clothing/suit/armor/vest/blueshirt/skyrat/orderly
-	r_pocket = /obj/item/assembly/flash/handheld
-	l_pocket = /obj/item/restraints/handcuffs/cable/blue
+	glasses = /obj/item/clothing/glasses/hud/health/sunglasses
```

Comment:
Also technically undocumented, but, sure

Reply:
Thank you for reminding me